projects/igniteui-angular/src/lib/date-range-picker/date-range-picker.component.ts

```diff
@@ -73,7 +78,7 @@ export class IgxDateRangePickerComponent extends DisplayDensityBase
      * ```
      */
     @Input()
-    public mode = InteractionMode.Dialog;
+    public mode = InteractionMode.DropDown;
```

Comment:
this is a breaking change!

Reply:
Please update and date range picker dev sample, as now there is no single one in dialog mode